app/components/add-layer/component.js

```diff
@@ -55,7 +55,6 @@ export default Ember.Component.extend({
                 case 'advanced': // Update when link is ready
```

Comment:
Yeah all these comments are from when link was not a layer I think. Removing them now.